cmd/drand-cli/public.go

```diff
@@ -8,9 +8,10 @@ import (
 	"os"
 
 	"github.com/drand/drand/chain"
+	"github.com/drand/drand/client/grpc"
+	clientinterface "github.com/drand/drand/client/interface"
```

Comment:
Im not a fan of this ... we should just have client. That starts down the path to have too many packages for doing one thing..
```
github.com/drand/drand/client/grpc
```
should be all what we need in this case, no ?

Reply:
since a normal client library won't be connecting over gRPC, having it split to a separate package means that when e.g. filecoin links to `drand/client` it doesn't take on gRPC dependencies.

Reply:
the interface is used for the result type

Reply:
Sorry I was wrong; i meant:
```
github.com/drand/drand/client/grpc
github.com/drand/drand/client/
```